packages/storybook/config/storybook/preview-head.html

```diff
@@ -13,58 +13,4 @@
     body {
         font-feature-settings: 'tnum'; /* Enable tabular numbers */
     }
-
-    h1.sbdocs-h1:first-of-type {
-        margin-bottom: 16px;
-    }
-
-    h2.sbdocs-h2,
-    h2.sbdocs-h2:first-of-type {
-        border-top: 1px solid #c5ccd3;
-        padding-top: 8px;
-        padding-bottom: 8px;
-    }
-
-    .sbdocs-content .sbdocs-a {
-        color: #125fca;
-        text-decoration: underline;
-    }
-    .sbdocs-content .sbdocs-a:hover {
-        color: #0f4fa9;
-    }
-
-    .os-content {
-        background-color: #333;
-    }
-
-    .os-viewport .os-content pre code {
-        color: #fff;
-    }
-
-    .sbdocs.sbdocs-p code,
```

Comment:
I can see some of these styles used on [pages such as the typography guide](https://vue.pie.design/?path=/story/documentation-guides-typography--page).

Reply:
thanks for the spot, was my bad, understanding was this just applied styles to the docs layout, I will put all these back and we can remove if needed.

Reply:
I pulled the branch and can see that now navigation paddings and font-size changed to be smaller. I can't see straight away what was removed/added to make this change but we need to fix it

Reply:
after our session yesterday going through the code I have updated styles accordingly.